Source/Scene/MetadataGroup.js

```diff
@@ -16,8 +16,6 @@ import MetadataEntity from "./MetadataEntity.js";
  *
```

Comment:
Above, remove `Implements the {@link MetadataEntity} interface` since this class is now public but `MetadataEntity` is not. Maybe that note could become a comment.

Reply:
Same with `MetadataTileset`